app/views/shared/_site_search_form.html.erb

```diff
@@ -4,15 +4,7 @@
     <label for="search_field" class="sr-only"><%= t('blacklight.search.form.search_field.label') %></label>
   <% end %>
   <div class="input-group">
-    <% if search_fields.length > 1 %>
-        <%= select_tag(:search_field,
-                       options_for_select(search_fields, h(params[:search_field])),
-                       title: t('blacklight.search.form.search_field.title'),
-                       id: "search_field",
-                       class: "custom-select search-field") %>
-    <% elsif search_fields.length == 1 %>
-      <%= hidden_field_tag :search_field, search_fields.first.last %>
-    <% end %>
+    <%= hidden_field_tag :search_field, search_fields.first.last %>
```

Comment:
This means that if we decide to re-add a search fields drop-down we'll want to pull this back. If we do that we would need to figure out how to inject our search fields configurations into various Spotlight controller contexts, which seems like a recipe for disaster (so this seems like a simple solution for now).